sandbox/ashley_user/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""ashley_user is a django application that defines a custom user model implementation."""
+
+# pylint: disable=invalid-name
+default_app_config = "ashley_user.apps.AshleyUserConfig"
```

Comment:
maybe ashley_user is not a good name. You can see it as the app in which we customize ashley for the project. I think the common practice in Django is to call it sandbox as well. @jmaupetit WDYT?